src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -4,6 +4,7 @@
 .NavSearchBar {
   position: relative;
   width: $search-dropdown-width;
+  background-color: inherit;
```

Comment:
As asked in other comments in this review, why do we need this `inherit` here? Is the default 'not set' state of 'transparent' (thus showing the element behind it) not sufficient?

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1350/commits/b6e481c7aad3433f43e567cdd0244c43fccfc78f